### PR TITLE
Correct command path and pincode interpolation

### PR DESCRIPTION
**Pull request by daemonsy**
## What's up? 

Hi there, was trying the example commands. Kept getting `405`, looked into the `device_unlock` example and realized the path is probably outdated. Rest of the changes in line comments.

### CODE REVIEW

File: tools/device_lock [hunk 1]
```diff
@@ -4,13 +4,13 @@
 #          ./tools/device_lock $udid $pincode
 #
 source $MDMDIRECTOR_ENV_PATH
-endpoint="device_lock"
+endpoint="device/command/device_lock"
 jq -n \
   --arg udid "$1" \
-  --arg pincode "$2"
+  --arg pincode "$2" \
```
Comment (daemonsy) [inline tools/device_lock:10]: Missing `\` broke the command

Comment (daemonsy) [inline tools/device_lock:10]: Missing `\` broke the command

File: tools/device_lock [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
   '.udids = [$udid]
   |.value = true
   |.push_now = true
-  |.pin = "[$pincode]"
```
Comment (daemonsy) [inline tools/device_lock:14]: This interpolated to `"pin": "[$pincode]"`, think that's probably not the intent.  With this change, `"pin": "123"`

**Review by discentem — APPROVED** [review]

**Comment by daemonsy** [comment]
Thanks for the approval @discentem, what else is needed to get this merged?

**Comment by discentem** [comment]
Nothing! Just forgot to click the button 😆

**Pull request merged**